petab/v2/core.py

```diff
@@ -745,13 +884,60 @@ def _validate_id(cls, v):
             raise ValueError(f"Invalid ID: {v}")
         return v
 
+    @field_validator("estimate", mode="before")
+    @classmethod
+    def _validate_estimate_before(cls, v):
+        if isinstance(v, bool):
+            return v
+
+        if isinstance(v, int) or isinstance(v, float) and v.is_integer():
+            if v == 0:
+                return False
+            if v == 1:
+                return True
+
+        if isinstance(v, str):
+            if v == "0":
+                return False
+            if v == "1":
+                return True
```

Comment:
Sufficient? Could also add `True` and `False` to remove the above code too...
```suggestion
        if v in [0, 1, "0", "1"]:
            return bool(int(v))
```

Reply:
I think that should work. But meanwhile https://github.com/PEtab-dev/PEtab/pull/614 happened. Needs updating.

Extra whitespace is not allowed, except for math expressions, right?

Reply:
> Extra whitespace is not allowed, except for math expressions, right?

Sounds fine to me. I don't see an issue either way, so whichever is more convenient

Reply:
Same here. To be discussed at https://github.com/PEtab-dev/PEtab/.